tests/istio.mk

```diff
@@ -84,8 +84,12 @@ e2e_bookinfo: istioctl generate_yaml
 e2e_upgrade: istioctl generate_yaml
 	go test -v -timeout 20m ./tests/e2e/tests/upgrade -args ${E2E_ARGS} ${EXTRA_E2E_ARGS}
 
+JUNIT_E2E_XML ?= $(ISTIO_OUT)/junit.xml
```

Comment:
Well, can't be just junit.xml - make it junit_e2e_all.xml or something.

And I think it'll be good to have each separate step generate an xml too - since we want
to run them in parallel ( at least in some CIs )

Reply:
Done